website/docs/integrations/zoho-flow.md

```diff
@@ -0,0 +1,68 @@
+---
+id: zoho-flow
+title: Zoho Flow
+---
+## Connect ConfigCat to 500+ apps with Zoho Flow
+
+<a href="https://www.zoho.com/flow/apps/configcat/integrations" target="_blank">ConfigCat's Zoho Flow integration</a> helps you connect ConfigCat to 500+ other web apps without any code. It helps you automate day-to-day tasks and build complex business workflows within minutes.
+
+You can set **triggers**, add **actions**, use **Delays and Decisions** to create your entire workflow on an easy to use builder.
+
+<a href="https://www.zoho.com/flow/apps/configcat/integrations" target="_blank">ConfigCat's Zoho Flow integration</a> can notify you about Feature Flags or Settings changes in ConfigCat, so you can configure e.g. with Slack to send a message about the changes applied in the ConfigCat environment.
+
+
+## Setup
+
+1. Log in to your <a href="https://www.zoho.com/flow/" target="_blank">Zoho Flow account</a> or create a new one.
+
+1. Navigate to `My Flows` on the side menu bar.
```

Comment:
1. Navigate to `My Flows` using the side menu.

Reply:
Done